CRM/Multicompanyaccounting/Hook/ValidateForm/FinancialTypeAccount.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * Form Validation on editing or assign a financial account for a type
+ */
+class CRM_Multicompanyaccounting_Hook_ValidateForm_FinancialTypeAccount {
+
+  private $form;
+  private $fields;
+  private $errors;
+
+  public function __construct(&$form, &$fields, &$errors) {
+    $this->form = $form;
+    $this->fields = &$fields;
+    $this->errors = &$errors;
+  }
+
+  /**
+   * Validates the financial type account form
```

Comment:
The should be the description of `validateFinancialAccountOrangistion` method not this one, given the `validate` method might contain more type of validations related to Financial accounts in the future.